Extensions/classic_tags.js

```diff
@@ -9,6 +9,7 @@ XKit.extensions.classic_tags = new Object({
 
 	running: false,
 	slow: true,
+	apiKey: XKit.api_key, //*"fuiKNFp9vQFvjLNvx4sUwti4Yb5yGutBN4Xh10LXZhhRKjWlV4",**//
```

Comment:
Done! I saw it in so many other places I actually thought it was supposed to be there.